source/2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Countdown to The New Year: Ember CLI Mirage
```

Comment:
Do you think it makes to stick to the lower case style of addons in this series? E.g.

```suggestion
title: Countdown to The New Year: ember-cli-mirage
```

What are your thoughts?

Reply:
not in the title of the article, but when referenced inside the article, yes.